MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+* Set the **path** option to the location of your logs on your nodes.
+* If you are using the **exclude_path** option from SCK, you can use the **excludePaths** option in Splunk OpenTelemetry Collector for Kubernetes to exclude any logs you want.
+* If you are using the **"logs"** option to define **"multiline**" configs in SCK, you can use the **multilineConfigs** option in Splunk OpenTelemetry Collector for Kubernetes to concatenate multiline logs. 
+* If you are using the **checkpointFile** option in SCK to define a custom location for checkpointing your logs, you can also do so with the **checkpointPath** option in Splunk OpenTelemetry Collector for Kubernetes.
+* If you are using the **"logs"** option to ingest any other logs from your nodes, you can also do so with the **extraFileLogs** option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Cluster Name
+If you are using the clusterName option to set a cluster name metadata field in your logs in SCK, you can also use the clusterName option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Metadata
+If you are using the customMetadata option to set a custom metadata field in your logs in SCK, you can use the customMetadata option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Scheduling configs
+If you are using any pod scheduling operations such as nodeSelector, affinity and tolerations in SCK, you can use the nodeSelector, affinity and tolerations options in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Service Accounts
+If you are using the serviceAccount option to use your own service accounts in SCK, you can use the serviceAccount option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Secrets
+If you are using the secret option to use your own service accounts in SCK, you can use the secret option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Custom docker image and pull secrets
+If you are using custom docker images, tags, pull secrets and pull policy in SCK, you can achieve the same using the "image" option in Splunk OpenTelemetry Collector for Kubernetes to specify the relevant configs for using docker images.
+
+#### Limiting/Increasing Application Resources
+If you are using the resources option in SCK to limit/increase the CPU and memory usage in SCK, you can do the same using the resources option in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Extra Files from Host
```

Comment:
"Extra files from the host"

"For tailing files other than container or journald logs (that is , kube audit logs), configure `extraFileLogs` using this [filelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) receiver configuration.